<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>450</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:600</string>
   </property>
   <property name="text">
    <string>시스템 메시지</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:600</string>
   </property>
   <property name="text">
    <string>재고</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:600</string>
   </property>
   <property name="text">
    <string>금액</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;
border-width: 1px;
border-style:outset;
min-width:1%;
min-height:2%;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stock">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;
border-width: 1px;
border-style:outset;
min-width:1%;
min-height:2%;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cost">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>380</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;
border-width: 1px;
border-style:outset;
min-width:1%;
min-height:2%;</string>
   </property>
   <property name="text">
    <string>000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:600</string>
   </property>
   <property name="text">
    <string>잔고</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_wallet">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>300</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;
border-width: 1px;
border-style:outset;
min-width:1%;
min-height:2%;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_purchase">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>300</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:600</string>
   </property>
   <property name="text">
    <string>구입</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_insert">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>350</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:600</string>
   </property>
   <property name="text">
    <string>입금</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>350</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:600</string>
   </property>
   <property name="text">
    <string>잔돈 반환</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_director">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:600</string>
   </property>
   <property name="text">
    <string>관리자 모드</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_system">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_name5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:10pt; font-weight:600</string>
   </property>
   <property name="text">
    <string>밀키스</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_drink7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>drink_images/7.png</normaloff>
     <normalon>drink_images/7_c.png</normalon>drink_images/7.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>85</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_name6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:10pt; font-weight:600</string>
   </property>
   <property name="text">
    <string>사이다</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_name1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:10pt; font-weight:600</string>
   </property>
   <property name="text">
    <string>코카콜라</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_name3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:10pt; font-weight:600</string>
   </property>
   <property name="text">
    <string>게토레이</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_drink3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>drink_images/3.png</normaloff>
     <normalon>drink_images/3_c.png</normalon>drink_images/3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>85</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_drink6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>drink_images/6.png</normaloff>
     <normalon>drink_images/6_c.png</normalon>drink_images/6.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>85</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_cost3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>가격</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_drink1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>drink_images/1.png</normaloff>
     <normalon>drink_images/1_c.png</normalon>drink_images/1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>85</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_cost5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>가격</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_drink2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>drink_images/2.png</normaloff>
     <normalon>drink_images/2_c.png</normalon>drink_images/2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>85</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_drink4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>drink_images/4.png</normaloff>
     <normalon>drink_images/4_c.png</normalon>drink_images/4.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>85</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_cost2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>가격</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_name0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:10pt; font-weight:600</string>
   </property>
   <property name="text">
    <string>펩시</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cost4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>가격</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cost1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>가격</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_drink0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>drink_images/0.png</normaloff>
     <normalon>drink_images/0_c.png</normalon>drink_images/0.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>85</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_name7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:10pt; font-weight:600</string>
   </property>
   <property name="text">
    <string>스파클링</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cost6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>가격</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_drink5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>drink_images/5.png</normaloff>
     <normalon>drink_images/5_c.png</normalon>drink_images/5.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>85</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_name2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:10pt; font-weight:600</string>
   </property>
   <property name="text">
    <string>코코팜</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cost7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>320</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>가격</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cost0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>가격</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_name4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:10pt; font-weight:600</string>
   </property>
   <property name="text">
    <string>레쓰비</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>531</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100,100,100)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_19</zorder>
  <zorder>label_20</zorder>
  <zorder>label_name</zorder>
  <zorder>label_stock</zorder>
  <zorder>label_cost</zorder>
  <zorder>label_3</zorder>
  <zorder>label_wallet</zorder>
  <zorder>pushButton_purchase</zorder>
  <zorder>pushButton_insert</zorder>
  <zorder>pushButton_return</zorder>
  <zorder>pushButton_director</zorder>
  <zorder>textEdit_system</zorder>
  <zorder>label_name5</zorder>
  <zorder>pushButton_drink7</zorder>
  <zorder>label_name6</zorder>
  <zorder>label_name1</zorder>
  <zorder>label_name3</zorder>
  <zorder>pushButton_drink3</zorder>
  <zorder>pushButton_drink6</zorder>
  <zorder>label_cost3</zorder>
  <zorder>pushButton_drink1</zorder>
  <zorder>label_cost5</zorder>
  <zorder>pushButton_drink2</zorder>
  <zorder>pushButton_drink4</zorder>
  <zorder>label_cost2</zorder>
  <zorder>label_name0</zorder>
  <zorder>label_cost4</zorder>
  <zorder>label_cost1</zorder>
  <zorder>pushButton_drink0</zorder>
  <zorder>label_name7</zorder>
  <zorder>label_cost6</zorder>
  <zorder>pushButton_drink5</zorder>
  <zorder>label_name2</zorder>
  <zorder>label_cost7</zorder>
  <zorder>label_cost0</zorder>
  <zorder>label_name4</zorder>
 </widget>
 <tabstops>
  <tabstop>pushButton_purchase</tabstop>
  <tabstop>pushButton_insert</tabstop>
  <tabstop>pushButton_director</tabstop>
  <tabstop>pushButton_return</tabstop>
  <tabstop>textEdit_system</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
